web/src/routes/Events.jsx

```diff
@@ -573,20 +587,52 @@ export default function Events({ path, ...props }) {
-                              options={{
-                                preload: 'auto',
-                                autoplay: true,
-                                sources: [
-                                  {
-                                    src: `${apiHost}vod/event/${event.id}/master.m3u8`,
-                                    type: 'application/vnd.apple.mpegurl',
-                                  },
-                                ],
-                              }}
-                              seekOptions={{ forward: 10, backward: 5 }}
-                              onReady={() => {}}
-                            />
+                            <div>
+                              <TimelineSummary
+                                event={event}
+                                onFrameSelected={(frame) => onEventFrameSelected(event, frame)}
+                              />
+                              <div>
+                                <VideoPlayer
+                                  options={{
+                                    preload: 'auto',
+                                    autoplay: true,
+                                    sources: [
+                                      {
+                                        src: `${apiHost}vod/event/${event.id}/master.m3u8`,
+                                        type: 'application/vnd.apple.mpegurl',
+                                      },
+                                    ],
+                                  }}
+                                  seekOptions={{ forward: 10, backward: 5 }}
+                                  onReady={(player) => {
+                                    this.player = player;
+                                    this.player.on('playing', () => {
+                                      setEventOverlay(undefined);
+                                    });
+                                  }}
+                                  onDispose={() => {
+                                    this.player = null;
+                                  }}
+                                >
+
+                                  {eventOverlay ? (
+                                    <div
+                                      className="absolute border-4 border-red-600"
+                                      style={{left: `${Math.round(
+                                        eventOverlay.data.box[0] * 100
+                                      )}%`, top: `${Math.round(eventOverlay.data.box[1] * 100)}%`, right: `${Math.round(
+                                        (1 - eventOverlay.data.box[2]) * 100
+                                      )}%`, bottom: `${Math.round((1 - eventOverlay.data.box[3]) * 100)}%`}}
+                                    >
+                                      {eventOverlay.class_type == 'entered_zone' ? (
+                                        <div className="absolute w-2 h-2 bg-red-600 left-[50%] bottom-0" />
```

Comment:
lets make the box for the bottom center green or yellow instead so its more obvious